kintone/cli-kintone
# build: migrate to pnpm

## Why
Yarn V1 is entered maintenance mode in January 2020 and will eventually reach their end-of-life in terms of support.
We want to migrate our projects to pnpm package.


## What

- Migrate from Yarn to pnpm.



## How to test
- CI result is all green after migration.
- Check script in package.json run normally


## Checklist
